tests/Commons.Persistence.EntityFramework.Tests/Generic/EntityDaoTests.cs

```diff
@@ -51,5 +53,24 @@ public void TestLoadByBusinessId()
                 actualEntity.Should().BeEquivalentTo(expectedEntity);
             }
         }
+
+        [Test]
+        public async Task TestLoadByBusinessIdAsync()
+        {
+            DbContextOptions<TestDbContext> contextOptions = GetDbContextOptions<TestDbContext>();
+            EntityWithStringKey expectedEntity = new EntityWithStringKey("Eine ID");
+            using (TestDbContext context = new TestDbContext(contextOptions))
```

Comment:
Der Testaufbau (GIVEN-Teil) ist in allen Tests der selbe und der Code entsprechend doppelt. Bitte die Redundanz entfernen und als in private wiederverwendbare Methode auslagern.